pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/configuration/RunConfiguration.kt

```diff
@@ -55,6 +57,13 @@ class RunConfiguration(
             logger.debug("MutationMateRunConfiguration: classFQN was updated to '$classFQN'.")
         }
 
+    override fun writeExternal(element: Element) {
+        super.writeExternal(element)
+        JDOMExternalizerUtil.writeField(element, "TASK_NAME", taskName)
+        JDOMExternalizerUtil.writeField(element, "GRADLE_EXECUTABLE", gradleExecutable)
+        JDOMExternalizerUtil.writeField(element, "CLASS_FQN", classFQN)
+    }
+
```

Comment:
Works but this seems like a weird work around.
What did work for me is setting the propertyName of the StoredProperty delegate in the RunConfigurationOptions correctly like so:

```
private var taskNameOption: StoredProperty<String?> = string("").provideDelegate(this, "taskName")
```

Reply:
I have added the respective names here: b07053b9d3f9f8e1a0a6b3f43a0ac36d3bbbba4f